web/src/views/public/reciters/Show.vue

```diff
@@ -141,4 +153,8 @@ export default {
 .title {
   margin-bottom: 12px;
 }
+
+.v-pagination {
```

Comment:
Looks like on the reciters card we are adding 16px on all sides and the all reciters page looks nice so i made the same change to make it look same on albums page